staging/cse/windows/azurecnifunc.ps1

```diff
@@ -41,56 +41,54 @@ function Set-AzureCNIConfig
         [Parameter(Mandatory=$true)][bool]
         $IsDualStackEnabled
     )
-    # Fill in DNS information for kubernetes.
-    if ($IsDualStackEnabled){
-        $subnetToPass = $KubeClusterCIDR -split ","
-        $exceptionAddresses = @($subnetToPass[0])
-    }
-    else {
-        $exceptionAddresses = @($KubeClusterCIDR)
-    }
-    $vnetCIDRs = $VNetCIDR -split ","
-    foreach ($cidr in $vnetCIDRs) {
-        $exceptionAddresses += $cidr
-    }
-
     $fileName  = [Io.path]::Combine("$AzureCNIConfDir", "10-azure.conflist")
     $configJson = Get-Content $fileName | ConvertFrom-Json
     $configJson.plugins.dns.Nameservers[0] = $KubeDnsServiceIp
     $configJson.plugins.dns.Search[0] = $KubeDnsSearchPath
 
-    $osBuildNumber = (get-wmiobject win32_operatingsystem).BuildNumber
-    if ($osBuildNumber -le 17763){
-        # In WS2019 and below rules in the exception list are generated by dropping the prefix lenght and removing duplicate rules.
-        # If multiple execptions are specified with different ranges we should only include the broadest range for each address.
-        # This issue has been addressed in 19h1+ builds
-
-        $processedExceptions = GetBroadestRangesForEachAddress $exceptionAddresses
-        Write-Log "Filtering CNI config exception list values to work around WS2019 issue processing rules. Original exception list: $exceptionAddresses, processed exception list: $processedExceptions"
-        $configJson.plugins.AdditionalArgs[0].Value.ExceptionList = $processedExceptions
-    }
-    else {
-        $configJson.plugins.AdditionalArgs[0].Value.ExceptionList = $exceptionAddresses
-    }
-
-    if ($IsDualStackEnabled){
-        $configJson.plugins[0]|Add-Member -Name "ipv6Mode" -Value "ipv6nat" -MemberType NoteProperty
-        $serviceCidr = $KubeServiceCIDR -split ","
-        $configJson.plugins[0].AdditionalArgs[1].Value.DestinationPrefix = $serviceCidr[0]
+    if ($global:IsDisableWindowsOutboundNat) {

```

Comment:
Can you help to add some unit test in `staging/cse/windows/azurecnifunc.tests.ps1` to make sure that the generated json file is expected?

Reply:
Thank you. Added unit tests.